styles/aboutUs.scss

```diff
@@ -3,81 +3,123 @@
   width: 100%;
   width: 100vw;
 
-  padding-top: 100px;
-  margin-bottom: 100px;
+  padding-top: $spc-l;
```

Comment:
Nice. When you have multiple paddings or margins for the same element shorthand is handy and better to use, for example:
`padding: $spc-l 0 $spc-xl 0;`